content/participate/document.adoc

```diff
@@ -55,7 +55,7 @@ You can open any GitHub Markdown or Asciidoc page in any Jenkins repo, click _Ed
 Useful links:
 
 * link:https://issues.jenkins-ci.org/issues/?filter=18650&jql=(project%20%3D%20WEBSITE%20or%20labels%20in%20(documentation%2C%20docs-sig))%20AND%20labels%20%3D%20newbie-friendly%20and%20status%20in%20(Open%2C%20Reopened%2C%20%22To%20Do%22)[Newbie-friendly documentation tasks]
-* https://issues.jenkins-ci.org/issues/?filter=18650&jql=((project%20%3D%20WEBSITE%20and%20component%20%3D%20content)%20or%20labels%20in%20(documentation%2C%20docs-sig))%20and%20status%20in%20(Open%2C%20Reopened%2C%20%22To%20Do%22)[More advanced documentation tasks] (no `newbie-friendly` label set)
+* link:https://issues.jenkins-ci.org/issues/?filter=18650&jql=project%20%3D%20WEBSITE%20and%20component%20%3D%20content%20or%20labels%20in%20(documentation%2C%20docs-sig)%20and%20status%20in%20(Open%2C%20Reopened%2C%20%22To%20Do%22)[More advanced documentation tasks] (no `newbie-friendly` label set)
```

Comment:
Could we use this as the excuse to switch from Jira for friendly issues to GitHub?  There are only two issues in Jira that are friendly for new documentation contributors and not already "in progress", while there are 6 GitHub issues that are friendly.  There will be more friendly issues in the future on GitHub as we continue to transition away from Jira for jenkins.io issues.

```suggestion
* link:https://github.com/jenkins-infra/jenkins.io/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22[More advanced documentation tasks] (no `newbie-friendly` label set)
```